client/views/teams/contextualBar/TeamChannelItem.js

```diff
@@ -107,7 +109,7 @@ export const TeamChannelItem = ({ room }) => {
 				<RoomAvatar room={room} size='x28' />
 			</Option.Avatar>
 			<Option.Column>{room.t === 'c' ? <Icon name='hash' size='x15'/> : <Icon name='hashtag-lock' size='x15'/>}</Option.Column>
-			<Option.Content>{room.fname || room.name}</Option.Content>
+			<Option.Content><Box display='inline-flex'>{room.fname || room.name} {room.teamDefault ? <Breadcrumbs.Tag>{t('Team_Auto-join')}</Breadcrumbs.Tag> : ''}</Box></Option.Content>
```

Comment:
Better to replace `room.fname || room.name` with `roomTypes.getRoomName(room.t, room)`.

Reply:
ok...